protocol/notifications.proto

```diff
@@ -0,0 +1,320 @@
+syntax = "proto3";
+package river;
+option go_package = "github.com/river-build/river/core/node/protocol";
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+service AuthenticationService {
```

Comment:
the AuthenticationService service isn't namespaced. Is that intentional? Will it serve as auth for every service going forward or is it just for the NotificationService?

Reply:
It is just for the nameservice although it's generic by nature. The session token is a JWT with the account id as the subject and the issuer and audience set to `ns`. But that can be changed if there is another service that requires authentication. Including switching over to a asymmetric signing schema that allows any service to validate the JWT.